src/libaktualizr/primary/sotauptaneclient.h

```diff
@@ -58,6 +58,7 @@ class SotaUptaneClient {
   bool isInstallCompletionRequired();
   void completeInstall();
   Uptane::LazyTargetsList allTargets();
+  Uptane::Target getCurrent() { return package_manager_->getCurrent(); }
```

Comment:
This is fine for now, but going forward it is likely we'll want something like this that is ECU-agnostic, i.e. doesn't just assume the primary. I just wanted to give you a heads-up so you're not surprised if/when that happens!